### ISSUE TITLE
Not compatible with the version IntelliJ IDEA 2023.2.3

### ISSUE THREAD

**Issue by JackieCheung** [axaluss/power-mode-intellij-plugin#65]

**Comment by MrClyfar** [comment]
Just updated Rider to v2023.2.3 and got the notification that Power Mode 2 is not compatible :(

Ahhh such a shame, I already miss the effects! lol

**Comment by MrClyfar** [comment]
Would love for this to be made compatible with the latest IDE version. I love this extension, it is so cool.

**Comment by MrClyfar** [comment]
Forgot to mention, I have Power Mode II installed in Jetbrains Rider

```
JetBrains Rider 2023.2.3
Build #RD-232.10203.29, built on November 1, 2023
Runtime version: 17.0.8.1+7-b1000.32 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.

```
and the plugin still works fine. I assume it just needs a re-compile and upload, so that the Marketplace site knows it's compatible?

**Comment by axaluss** [comment]
should be fixed in [Release 103.301](https://github.com/axaluss/power-mode-intellij-plugin/releases/tag/103.301)
